packages/react-jsx-highcharts/src/components/Tooltip/Tooltip.js

```diff
@@ -13,7 +13,7 @@ class Tooltip extends Component {
   };
 
   static defaultProps = {
-    ...Highcharts.defaultOptions.tooltip,
+    ...(Highcharts.defaultOptions && Highcharts.defaultOptions.tooltip),
```

Comment:
Fixed the [RangeSelector](/whawker/react-jsx-highcharts/blob/master/packages/react-jsx-highstock/src/components/RangeSelector/RangeSelector.js#L16) too.

I haven't tested `Highcharts.getOptions()`, but since this method is defined inside the same factory function, it probably won't work.